src/site/content/en/blog/more-capable-form-controls/index.md

```diff
@@ -0,0 +1,305 @@
+---
+title: "More capable form controls"
```

Comment:
Do you think it's worth highlighting that this relates to custom elements in the title?

Reply:
ah I see, it's actually talking about two things: `formdata` and custom elements. Maybe 'More capable form controls' is ok then. But it would still be nice to beef up the subhead if possible.

Reply:
Well, only one of the two APIs is limited to custom elements. That API takes up the bulk of the article, it's true... Perhaps I should add that detail in the subhead, instead?